### DISCUSSION TITLE
Tags Not Showing in PnP Modern Search Results

### DISCUSSION THREAD

**Discussion by hiddevdbroek** [microsoft-search/pnp-modern-search#3969] [General]
We are currently setting up a SharePoint site using PnP Modern Search and have encountered an issue with tags from the Term Store not appearing in search results. Specifically, when we tag documents using terms from the Term Store and attempt to search using the PnP Modern Search web parts, the documents do not show in the results.

This problem is impacting our ability to filter documents as intended, and any guidance on how to resolve this would be greatly appreciated.

See attached images for reference.
![2](https://github.com/user-attachments/assets/f0d8b1d6-1959-4fcf-ac55-9c064ac8a872)
![3](https://github.com/user-attachments/assets/6b744e23-cef3-475f-839a-d37889477dc5)
![4](https://github.com/user-attachments/assets/c9d96aad-a73a-4502-a2fe-ce34d669ad28)
![1](https://github.com/user-attachments/assets/4a0646d5-e70e-4f6f-90ed-c13ee7935ba8)

**Comment by kasperbolarsen** [comment]
Hi
In the resently released version 4.13.1 we have updated the way Tags are shown so I recommend updating to that version :-)
